test/unit/metrics/test_metricframe_smoke.py

```diff
@@ -226,6 +226,95 @@ def test_1m_1sf_1cf_metric_dict(transform_y_t, transform_y_p):
     assert target_maxs['recall']['m'] == max(recall_m_arr)
 
 
+def test_1m_1_sf_sample_weights():
+
+    def multi_sp(y_t, y_p, p1, p2):
+        assert len(y_t) == len(y_p)
```

Comment:
could you please add a comment about what this function does?

Reply:
Done